server.py

```diff
@@ -2,14 +2,14 @@
 from flask_restful import Resource, Api
 from flask_jsonpify import jsonify
 
-from services.dummy_function import get_chord_probabilities
+from services.chord_stats import getNextChord
 
 app = Flask(__name__)
 api = Api(app)
 
 class NextChord(Resource):
     def get(self, chord):
-        return jsonify(get_chord_probabilities(chord))
+        return jsonify(getNextChord([chord], depth=None))
```

Comment:
Instead of passing in a single chord, we probably want to pass in up to 3-4 chords. We could take in a string of up to 4 chords in the HTTP query and parse it into array for getNextChord to use.

Not high priority, but something to think about

Reply:
I'll have this fixed by tomorrow (3/7)